bindings.xjb

```diff
@@ -0,0 +1,78 @@
+<!--
```

Comment:
Suggestion: can this file be moved into the server directory? The example project at https://github.com/sbt/sbt-xjc/blob/master/src/sbt-test/sbt-xjc/xero/build.sbt puts both the xjb and xsd files under src/main/resources.